packages/device_info_plus/device_info_plus/lib/device_info_plus.dart

```diff
@@ -25,7 +25,8 @@ export 'src/model/macos_device_info.dart';
 export 'src/model/web_browser_info.dart';
 export 'src/model/windows_device_info.dart';
 
-export 'src/device_info_plus_linux.dart';
+export 'src/device_info_plus_linux.dart'
+    if (dart.library.html) 'src/device_info_plus_web.dart';
 export 'src/device_info_plus_windows.dart'
     if (dart.library.html) 'src/device_info_plus_web.dart';
```

Comment:
Can we chain two if clauses, instead of exporting the same library twice?

Reply:
No, unfortunately we can't. As far as I understand there are no issues with such export or, otherwise, we would see failing builds.

Reply:
Going to merge to release new versions in nearest time